### ISSUE TITLE
Switching to not memory efficient swish worsens model's performance

### ISSUE THREAD

**Issue by JohnSWD** [lukemelas/EfficientNet-PyTorch#254]
Hello,

I fine-tuned a model and now want to export it to ONNX. But I noticed that adding `model.set_swish(memory_efficient=False)` before applying the model to a test set results in a big drop in accuracy (from 0.83 to 0.7). Have anyone had this problem and found a solution to it?
